clients/admin-ui/src/types/api/models/SystemResponse.ts

```diff
@@ -0,0 +1,112 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+import type { ContactDetails } from "./ContactDetails";
+import type { DataFlow } from "./DataFlow";
+import type { DataProtectionImpactAssessment } from "./DataProtectionImpactAssessment";
+import type { DataResponsibilityTitle } from "./DataResponsibilityTitle";
+import type { PrivacyDeclarationResponse } from "./PrivacyDeclarationResponse";
+import type { SystemMetadata } from "./SystemMetadata";
+
+/**
+ * Extension of base pydantic model to include `privacy_declarations.id` fields in responses
+ */
+export type SystemResponse = {
+  /**
+   * A unique key used to identify this resource.
+   */
+  fides_key: string;
+  /**
+   * Defines the Organization that this resource belongs to.
+   */
+  organization_fides_key?: string;
+  tags?: Array<string>;
+  /**
+   * Human-Readable name for this resource.
+   */
+  name?: string;
+  /**
+   * A detailed description of what this resource is.
+   */
+  description?: string;
+  /**
+   * The id of the system registry, if used.
+   */
+  registry_id?: number;
+  /**
+   * An optional property to store any extra information for a system. Not used by fidesctl.
+   */
+  meta?: Record<string, string>;
+  /**
+   *
+   * The SystemMetadata resource model.
+   *
+   * Object used to hold application specific metadata for a system
+   *
+   */
+  fidesctl_meta?: SystemMetadata;
+  /**
+   * A required value to describe the type of system being modeled, examples include: Service, Application, Third Party, etc.
+   */
+  system_type: string;
+  /**
+   *
+   * The model defining the responsibility or role over
+   * the system that processes personal data.
+   *
+   * Used to identify whether the organization is a
+   * Controller, Processor, or Sub-Processor of the data
+   *
+   */
+  data_responsibility_title?: DataResponsibilityTitle;
+  /**
+   * The resources to which the System sends data.
+   */
+  egress?: Array<DataFlow>;
+  /**
+   * The resources from which the System receives data.
+   */
+  ingress?: Array<DataFlow>;
+  /**
+   * Extension of base pydantic model to include DB `id` field in the response
+   */
+  privacy_declarations: Array<PrivacyDeclarationResponse>;
```

Comment:
oh okay, the typing problem I [mentioned in the ts-ignore](https://github.com/ethyca/fides/blob/3054d348f7ddad93d4a953cbf83da77c12c768f7/clients/admin-ui/src/features/system/privacy-declarations/PrivacyDeclarationForm.tsx#L64-L66) might actually be that we should be using `SystemResponse`, not `System`. `System` has `privacy_declarations` listed without the id, while `SystemResponse`  has it with the `id`. so then the question is... what is `System`?? 😅 

in any case, that's too large of a change for this PR 👍